### PR TITLE
Fix br tag causing malformed document

### PR DESCRIPTION
**Pull request by skinofstars**
This effects new installs in some circumstances.

### CODE REVIEW

File: library/core/class.controller.php [hunk 1]
```diff
@@ -545,7 +545,7 @@ public function data($Path, $Default = '') {
     public function definitionList($wrap = true) {
         $Session = Gdn::session();
         if (!array_key_exists('TransportError', $this->_Definitions)) {
-            $this->_Definitions['TransportError'] = T('Transport error: %s', 'A fatal error occurred while processing the request.<br />The server returned the following response: %s');
+            $this->_Definitions['TransportError'] = T('Transport error: %s', 'A fatal error occurred while processing the request. The server returned the following response: %s');
```
Comment (DaazKu) [inline library/core/class.controller.php]: The <br> was probably there for a reason. It should be replaced by `\n` not space.

Comment (skinofstars) [inline library/core/class.controller.php]: Any line-break here seems presentationally unnecessary.

Comment (DaazKu) [inline library/core/class.controller.php]: `alert(definition('TransportError').replace('%s', status));` ?

Comment (DaazKu) [inline library/core/class.controller.php]: Also, what is the actual error? `$this->_Definitions` is passed through `json_encode` and I don't see how this is raising an error.

Comment (skinofstars) [inline library/core/class.controller.php]: It doesn't throw a PHP error, it returns a malformed HTML page that browsers can't display.

**Comment by DaazKu** [comment]
Could you be more specific on the circumstances that leads to give you an error?
I'm asking because I did a fresh install of the forum and going on the forum (with Chrome) I don't have any error.

**Pull request closed without merging**